bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -196,23 +196,18 @@ for i in range(2, len(insps)+1):
         coinctag = '{}det_'.format(len(ifocomb)) + ordered_ifo_list
         ctagcomb = [tag, 'full', coinctag]
 
-        final_bg_files[ordered_ifo_list] = wf.setup_multiifo_interval_coinc(workflow,
-                                                                            hdfbank,
-                                                                            inspcomb,
-                                                                            statfiles,
-                                                                            final_veto_file,
-                                                                            final_veto_name,
-                                                                            output_dir,
-                                                                            pivot_ifo,
-                                                                            fixed_ifo,
-                                                                            tags=ctagcomb)
+        bg_file = wf.setup_multiifo_interval_coinc(
+            workflow, hdfbank, inspcomb, statfiles, final_veto_file,
+            final_veto_name, output_dir, pivot_ifo, fixed_ifo, tags=ctagcomb)[0][1][0]
+
+        # Optionally perform follow-up on triggers and rerank the candidates
+        final_bg_files[ordered_ifo_list] = wf.rerank_coinc_followup(
+                       workflow, bg_file, hdfbank[0], output_dir,
+                       tags=ctagcomb)
 
 combctags = [tag]
-final_bg_file_list = wf.FileList()
-for key in final_bg_files:
-    final_bg_file_list += final_bg_files[key][0][1]
 combined_bg_file = wf.setup_multiifo_combine_statmap(workflow,
-                                                     final_bg_file_list,
+                                                     final_bg_files.values(),
```

Comment:
I think this should be a `wf.FileList`, why the change? We should be more careful about not sending lists of Files, but sending FileLists so that the functionality of the latter is available,

Reply:
Reason for the change: It was a bit confusing how there were multiple places carrying around the same information. So it removes a bit of redundancy.  However, you are right that it should probably be a FileList, so I've made it one now.